requirements.txt

```diff
@@ -41,7 +41,7 @@ pytest-mock==1.1
 typing==3.6.4
 
 
-amundsen-common==0.1.3
+amundsen-common==0.1.3rc0
```

Comment:
should it be rc0?

Reply:
Yes, 0.1.3 requires Python 3.7 where metadata service is not. `rc0` is the one with Python 3.6.

I think we can quickly get out of this situation when metadata service adopts amundsen-common==0.1.7 just released with Python 3.6